extension-manifest-v3/src/background/telemetry/index.js

```diff
@@ -0,0 +1,53 @@
+import { Metrics as Telemetry } from '@ghostery/libs';
+import { store } from 'hybrids';
+
+import Options from '/store/options.js';
+import UTMs from '/store/utms.js';
+
+import { loadStorage, saveStorage } from './storage';
```

Comment:
The `index.js` is still a quite small file, so I don't see a reason to take out those two functions into a separate file. Then you can back to the `background/telemetry.js` - which would contain the whole feature.

Reply:
done